content/en/network_monitoring/performance/setup.md

```diff
@@ -450,6 +440,21 @@ To set up on AWS ECS, see the [AWS ECS][1] documentation page.
 {{% /tab %}}
 {{< /tabs >}}
 
+### Enhanced Resolution
```

Comment:
Can you move the site-region partial above the title? When I toggle to AP1 or US1-FED, the whole section should be hidden.

```suggestion
{{< site-region region="us,us3,us5,eu" >}}
### Enhanced Resolution
```